clar2wasm/src/words/blockinfo.rs

```diff
@@ -402,7 +403,17 @@ mod tests {
 
         #[test]
         fn get_block_info_less_than_two_args() {
-            let mut env = TestEnvironment::default();
+            let epoch = if cfg!(feature = "test-clarity-v1") {
+                StacksEpochId::Epoch2_05
+            } else {
+                StacksEpochId::Epoch25
+            };
+
+            let mut env = TestEnvironment::new(
+                epoch,
+                ClarityVersion::default_for_epoch(epoch)
+            );
```

Comment:
Wait, that confuses me. Here, whatever the epoch chosen, the default Clarity version will be Clarity 2, no? Even when the feature "test-clarity-v1" is set.

Reply:
Nop. The default Clarity version for epoch 2.05 is Clarity::V1 and for epoch 2.5 is Clarity::V2.

Reply:
_Solved on slack_ versions are okay for the selected epochs.